2019-10-20 19:05:44 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'current user should either be set in ZCRMRestClient or in configuration  map' in C:\xampp\htdocs\zoho\vendor\zohocrm\php-sdk\src\crm\utility\ZCRMConfigUtil.php(93)
Trace::#0 C:\xampp\htdocs\zoho\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(71): zcrmsdk\crm\utility\ZCRMConfigUtil::getAccessToken()
#1 C:\xampp\htdocs\zoho\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(88): zcrmsdk\crm\api\APIRequest->authenticateRequest()
#2 C:\xampp\htdocs\zoho\vendor\zohocrm\php-sdk\src\crm\api\handler\OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#3 C:\xampp\htdocs\zoho\vendor\zohocrm\php-sdk\src\crm\setup\restclient\ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#4 C:\xampp\htdocs\zoho\rest.php(253): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#5 C:\xampp\htdocs\zoho\rest.php(287): RestC::getOrganizationDetails()
#6 {main}
2019-10-24 12:41:38 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid oauth scope to access this URL' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\APIResponse.php(93)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401, NULL)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\setup\restclient\ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#6 D:\morecraf\agentadmin\agentadmin\rest.php(266): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#7 D:\morecraf\agentadmin\agentadmin\rest.php(310): RestC::getOrganizationDetails()
#8 {main}
2019-10-24 12:42:10 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid oauth scope to access this URL' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\APIResponse.php(93)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401, NULL)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\setup\restclient\ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#6 D:\morecraf\agentadmin\agentadmin\rest.php(266): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#7 D:\morecraf\agentadmin\agentadmin\rest.php(310): RestC::getOrganizationDetails()
#8 {main}
2019-10-24 12:44:27 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid oauth scope to access this URL' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\APIResponse.php(93)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 401 \r\n...', 401, NULL)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 401 \r\n...', 401)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\setup\restclient\ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#6 D:\morecraf\agentadmin\agentadmin\rest.php(266): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#7 D:\morecraf\agentadmin\agentadmin\rest.php(310): RestC::getOrganizationDetails()
#8 {main}
2019-10-24 13:15:31 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'Update your TLS version from TLSv1 to TLSv1.2' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\APIResponse.php(93)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400, NULL)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\setup\restclient\ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#6 D:\morecraf\agentadmin\agentadmin\rest.php(266): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#7 D:\morecraf\agentadmin\agentadmin\rest.php(310): RestC::getOrganizationDetails()
#8 {main}
2019-10-24 13:15:57 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'Update your TLS version from TLSv1 to TLSv1.2' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\APIResponse.php(93)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400, NULL)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\setup\restclient\ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#6 D:\morecraf\agentadmin\agentadmin\rest.php(266): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#7 D:\morecraf\agentadmin\agentadmin\rest.php(310): RestC::getOrganizationDetails()
#8 {main}
2019-10-24 13:19:41 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'Update your TLS version from TLSv1 to TLSv1.2' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\APIResponse.php(93)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400, NULL)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\OrganizationAPIHandler.php(112): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\setup\restclient\ZCRMRestClient.php(132): zcrmsdk\crm\api\handler\OrganizationAPIHandler->getOrganizationDetails()
#6 D:\morecraf\agentadmin\agentadmin\rest.php(266): zcrmsdk\crm\setup\restclient\ZCRMRestClient::getOrganizationDetails()
#7 D:\morecraf\agentadmin\agentadmin\rest.php(310): RestC::getOrganizationDetails()
#8 {main}
2019-10-25 16:34:21 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'Please check if the URL trying to access is a correct one' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\APIResponse.php(93)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 404 \r\n...', 404, NULL)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 404 \r\n...', 404)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\EntityAPIHandler.php(38): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModule.php(1070): zcrmsdk\crm\api\handler\EntityAPIHandler->getRecord()
#6 D:\morecraf\agentadmin\agentadmin\rest.php(322): zcrmsdk\crm\crud\ZCRMModule->getRecord(4.0719930000004E+18)
#7 {main}
2019-10-25 16:34:48 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'Please check if the URL trying to access is a correct one' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\APIResponse.php(93)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\APIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\APIResponse.php(34): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 404 \r\n...', 404, NULL)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(97): zcrmsdk\crm\api\response\APIResponse->__construct('HTTP/1.1 404 \r\n...', 404)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\EntityAPIHandler.php(38): zcrmsdk\crm\api\APIRequest->getAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModule.php(1070): zcrmsdk\crm\api\handler\EntityAPIHandler->getRecord()
#6 D:\morecraf\agentadmin\agentadmin\rest.php(322): zcrmsdk\crm\crud\ZCRMModule->getRecord(4.0719930000004E+18)
#7 {main}
2019-10-30 10:53:46 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'Please check if the URL trying to access is a correct one' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(66)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 404 \r\n...', 404)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 404 \r\n...', 404)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\rest.php(353): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Accounts')
#8 {main}
2019-10-30 10:55:13 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'Please check if the URL trying to access is a correct one' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(66)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 404 \r\n...', 404)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 404 \r\n...', 404)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\rest.php(353): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Contacts')
#8 {main}
2019-10-30 10:57:41 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'Please check if the URL trying to access is a correct one' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(66)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 404 \r\n...', 404)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 404 \r\n...', 404)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\rest.php(353): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Contacts')
#8 {main}
2019-10-30 10:58:17 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'the relation name given seems to be invalid' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(66)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\rest.php(353): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Accounts')
#8 {main}
2019-10-30 10:58:38 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'the relation name given seems to be invalid' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(66)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\rest.php(353): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Leads')
#8 {main}
2019-10-30 11:00:06 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid query formed' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(66)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\MassEntityAPIHandler.php(337): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModule.php(1138): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords('Parent_Id.id:eq...', 1, 1, 'criteria')
#6 D:\morecraf\agentadmin\agentadmin\rest.php(322): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Parent_Id.id:eq...', 1, 1)
#7 {main}
2019-10-30 11:00:16 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid query formed' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(66)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\MassEntityAPIHandler.php(337): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModule.php(1138): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords('Parent_Id:equal...', 1, 1, 'criteria')
#6 D:\morecraf\agentadmin\agentadmin\rest.php(322): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Parent_Id:equal...', 1, 1)
#7 {main}
2019-10-30 11:07:52 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'invalid query formed' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(66)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 400 \r\n...', 400)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\MassEntityAPIHandler.php(337): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModule.php(1138): zcrmsdk\crm\api\handler\MassEntityAPIHandler->searchRecords('Parent_Id:equal...', 1, 1, 'criteria')
#6 D:\morecraf\agentadmin\agentadmin\rest.php(322): zcrmsdk\crm\crud\ZCRMModule->searchRecordsByCriteria('Parent_Id:equal...', 1, 1)
#7 {main}
2019-11-02 00:05:11 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\application\models\ZoHo_Account.php(390): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Contacts')
#8 D:\morecraf\agentadmin\agentadmin\application\controllers\Portal.php(35): ZoHo_Account->LoadAccount('407199300000041...')
#9 D:\morecraf\agentadmin\agentadmin\system\core\CodeIgniter.php(532): Portal->entity('407199300000041...')
#10 D:\morecraf\agentadmin\agentadmin\index.php(276): require_once('D:\\morecraf\\age...')
#11 {main}
2019-11-02 00:14:37 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\application\models\ZoHo_Account.php(390): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Contacts')
#8 D:\morecraf\agentadmin\agentadmin\application\controllers\Portal.php(36): ZoHo_Account->LoadAccount('407199300000041...')
#9 D:\morecraf\agentadmin\agentadmin\system\core\CodeIgniter.php(532): Portal->entity('407199300000041...')
#10 D:\morecraf\agentadmin\agentadmin\index.php(276): require_once('D:\\morecraf\\age...')
#11 {main}
2019-11-02 00:14:54 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\application\models\ZoHo_Account.php(390): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Contacts')
#8 D:\morecraf\agentadmin\agentadmin\application\controllers\Portal.php(36): ZoHo_Account->LoadAccount('407199300000041...')
#9 D:\morecraf\agentadmin\agentadmin\system\core\CodeIgniter.php(532): Portal->entity('407199300000041...')
#10 D:\morecraf\agentadmin\agentadmin\index.php(276): require_once('D:\\morecraf\\age...')
#11 {main}
2019-11-02 00:15:48 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\application\models\ZoHo_Account.php(390): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Contacts')
#8 D:\morecraf\agentadmin\agentadmin\application\controllers\Portal.php(36): ZoHo_Account->LoadAccount('407199300000041...')
#9 D:\morecraf\agentadmin\agentadmin\system\core\CodeIgniter.php(532): Portal->entity('407199300000041...')
#10 D:\morecraf\agentadmin\agentadmin\index.php(276): require_once('D:\\morecraf\\age...')
#11 {main}
2019-11-02 00:16:31 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\application\models\ZoHo_Account.php(392): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Contacts')
#8 D:\morecraf\agentadmin\agentadmin\application\controllers\Portal.php(36): ZoHo_Account->LoadAccount('407199300000041...')
#9 D:\morecraf\agentadmin\agentadmin\system\core\CodeIgniter.php(532): Portal->entity('407199300000041...')
#10 D:\morecraf\agentadmin\agentadmin\index.php(276): require_once('D:\\morecraf\\age...')
#11 {main}
2019-11-02 00:17:29 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\application\models\ZoHo_Account.php(392): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Contacts')
#8 D:\morecraf\agentadmin\agentadmin\application\controllers\Portal.php(36): ZoHo_Account->LoadAccount('407199300000041...')
#9 D:\morecraf\agentadmin\agentadmin\system\core\CodeIgniter.php(532): Portal->entity('407199300000041...')
#10 D:\morecraf\agentadmin\agentadmin\index.php(276): require_once('D:\\morecraf\\age...')
#11 {main}
2019-11-02 00:18:15 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\application\models\ZoHo_Account.php(392): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Contacts')
#8 D:\morecraf\agentadmin\agentadmin\application\controllers\Portal.php(36): ZoHo_Account->LoadAccount('407199300000041...')
#9 D:\morecraf\agentadmin\agentadmin\system\core\CodeIgniter.php(532): Portal->entity('407199300000041...')
#10 D:\morecraf\agentadmin\agentadmin\index.php(276): require_once('D:\\morecraf\\age...')
#11 {main}
2019-11-02 00:18:46 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\application\models\ZoHo_Account.php(392): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Contacts')
#8 D:\morecraf\agentadmin\agentadmin\application\controllers\Portal.php(36): ZoHo_Account->LoadAccount('407199300000041...')
#9 D:\morecraf\agentadmin\agentadmin\system\core\CodeIgniter.php(532): Portal->entity('407199300000041...')
#10 D:\morecraf\agentadmin\agentadmin\index.php(276): require_once('D:\\morecraf\\age...')
#11 {main}
2019-11-02 00:20:29 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\application\models\ZoHo_Account.php(392): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Contacts')
#8 D:\morecraf\agentadmin\agentadmin\application\controllers\Portal.php(36): ZoHo_Account->LoadAccount('407199300000041...')
#9 D:\morecraf\agentadmin\agentadmin\system\core\CodeIgniter.php(532): Portal->entity('407199300000041...')
#10 D:\morecraf\agentadmin\agentadmin\index.php(276): require_once('D:\\morecraf\\age...')
#11 {main}
2019-11-02 00:21:30 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\application\models\ZoHo_Account.php(392): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Contacts')
#8 D:\morecraf\agentadmin\agentadmin\application\controllers\Portal.php(36): ZoHo_Account->LoadAccount('407199300000041...')
#9 D:\morecraf\agentadmin\agentadmin\system\core\CodeIgniter.php(532): Portal->entity('407199300000041...')
#10 D:\morecraf\agentadmin\agentadmin\index.php(276): require_once('D:\\morecraf\\age...')
#11 {main}
2019-11-02 00:22:07 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\application\models\ZoHo_Account.php(392): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Contacts')
#8 D:\morecraf\agentadmin\agentadmin\application\controllers\Portal.php(36): ZoHo_Account->LoadAccount('407199300000041...')
#9 D:\morecraf\agentadmin\agentadmin\system\core\CodeIgniter.php(532): Portal->entity('407199300000041...')
#10 D:\morecraf\agentadmin\agentadmin\index.php(276): require_once('D:\\morecraf\\age...')
#11 {main}
2019-11-02 00:23:53 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\application\models\ZoHo_Account.php(392): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Contacts')
#8 D:\morecraf\agentadmin\agentadmin\application\controllers\Portal.php(36): ZoHo_Account->LoadAccount('407199300000041...')
#9 D:\morecraf\agentadmin\agentadmin\system\core\CodeIgniter.php(532): Portal->entity('407199300000041...')
#10 D:\morecraf\agentadmin\agentadmin\index.php(276): require_once('D:\\morecraf\\age...')
#11 {main}
2019-11-02 00:24:14 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\application\models\ZoHo_Account.php(392): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Contacts')
#8 D:\morecraf\agentadmin\agentadmin\application\controllers\Portal.php(36): ZoHo_Account->LoadAccount('407199300000041...')
#9 D:\morecraf\agentadmin\agentadmin\system\core\CodeIgniter.php(532): Portal->entity('407199300000041...')
#10 D:\morecraf\agentadmin\agentadmin\index.php(276): require_once('D:\\morecraf\\age...')
#11 {main}
2019-11-02 00:24:43 ERROR: zcrmsdk\crm\exception\ZCRMException Caused by:'No Content' in D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(61)
Trace::#0 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(76): zcrmsdk\crm\api\response\BulkAPIResponse->handleForFaultyResponses()
#1 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\CommonAPIResponse.php(67): zcrmsdk\crm\api\response\CommonAPIResponse->processResponse()
#2 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\response\BulkAPIResponse.php(47): zcrmsdk\crm\api\response\CommonAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#3 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\APIRequest.php(123): zcrmsdk\crm\api\response\BulkAPIResponse->__construct('HTTP/1.1 204 \r\n...', 204)
#4 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\api\handler\RelatedListAPIHandler.php(57): zcrmsdk\crm\api\APIRequest->getBulkAPIResponse()
#5 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMModuleRelation.php(108): zcrmsdk\crm\api\handler\RelatedListAPIHandler->getRecords(NULL, NULL, 1, 20)
#6 D:\morecraf\agentadmin\agentadmin\vendor\zohocrm\php-sdk\src\crm\crud\ZCRMRecord.php(560): zcrmsdk\crm\crud\ZCRMModuleRelation->getRecords(NULL, NULL, 1, 20)
#7 D:\morecraf\agentadmin\agentadmin\application\models\ZoHo_Account.php(392): zcrmsdk\crm\crud\ZCRMRecord->getRelatedListRecords('Contacts')
#8 D:\morecraf\agentadmin\agentadmin\application\controllers\Portal.php(36): ZoHo_Account->LoadAccount('407199300000041...')
#9 D:\morecraf\agentadmin\agentadmin\system\core\CodeIgniter.php(532): Portal->entity('407199300000041...')
#10 D:\morecraf\agentadmin\agentadmin\index.php(276): require_once('D:\\morecraf\\age...')
#11 {main}
